fastify.js

```diff
@@ -917,6 +914,24 @@ function fastify (options) {
     this[kGenReqId] = reqIdGenFactory(this[kOptions].requestIdHeader, func)
     return this
   }
+
+  function acceptHTTPMethod (method, { hasBody = false } = {}) {
+    if (typeof method !== 'string' || http.METHODS.indexOf(method) === -1) {
```

Comment:
I see that `http.METHODS` lists pretty much all of the WebDav methods. But what if someone wants to add some random method like `SILLY`? Is it possible to do so if we don't check against that list?

Reply:
Node.js blocks requests that are not found in `http.METHODS` before they even reach Fastify I believe

I'd tried adding similar functionality and that was the case

Reply:
I can confirm what @gurgunday said

Reply:
Very disappointing.